integration_tests/tests/consistency/consistency_events.sql

```diff
@@ -0,0 +1,47 @@
+{{ config(
```

Comment:
I got this error when trying to test this model in the integration_tests validation run:

```
10:02:09    Database Error in test consistency_events (tests/consistency/consistency_events.sql)
  Queries in EXCEPT DISTINCT have mismatched column count; query 1 has 73 columns, query 2 has 72 columns at [25:5]
```

This is likely because `experiment_id` now is being removed in the default version. It shouldn't be too difficult to test that it successfully ran though--can you try and document your process for a successful test here?

Reply:
Thanks for catching this-- I reran the tests and see the errors now. Actually, `experiment_id` should be there as default so this was unexpected. This led me to looking into the prod/dev runs and actually the missing column is `channel_type`. Hmm. I will continuing looking into why this is happening

![image](https://github.com/user-attachments/assets/5dfe159c-e5f8-42ee-9d9c-1713ca969f70)

Reply:
@fivetran-reneeli Strange. Can you look in the upstream models and see if they are present there?

Reply:
Thanks Avinash for helping troubleshoot and realize it was due to a missing comma!